### PR TITLE
Improve sparsity polynomial scheduling

### PR DESCRIPTION
**Pull request by vshampor**

### CODE REVIEW

File: nncf/config_schema.py [hunk 1]
```diff
@@ -322,20 +322,28 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
                                             "of scheduler steps."),
     "power": with_attributes(_NUMBER,
                              description="For polynomial scheduler - determines the corresponding power value."),
+    "concave": with_attributes(_BOOLEAN, description="For polynomial scheduler - if True, then the target sparsiy level"
```
Comment (AlexKoff88) [inline nncf/config_schema.py:325]: `sparsiy` -> `sparsity`

File: nncf/config_schema.py [hunk 2]
```diff
@@ -322,20 +322,28 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
...
+                                                     "will be aproach in concave manner, and in convex manner "
```
Comment (AlexKoff88) [inline nncf/config_schema.py:326]: `aproach` -> `approached`

File: nncf/config_schema.py (hunk 1 above)
Comment (vshampor) [reply nncf/config_schema.py]: Done

File: nncf/config_schema.py (hunk 2 above)
Comment (vshampor) [reply nncf/config_schema.py]: Done

**Pull request merged**